docs/topics/request-response.rst

```diff
@@ -316,6 +317,13 @@ download_timeout
 The amount of time (in secs) that the downloader will wait before timing out.
 See also: :setting:`DOWNLOAD_TIMEOUT`.
 
+.. reqmeta:: download_latency
+
+download_latency
+----------------
+
+The amount of time spent to fetch the response, since the request has been
+started.
```

Comment:
I updated, according to @kmike's observations. What about the moment when the requests are considered started?